Release 2.4.0: Update changelog & shared to v170 (#158)
NordicSemiconductor/pc-nrfconnect-dtm: Changelog.md

## 2.4.0 - 2024-04-08

### Changed

> -   Moved feedback tab to a dialog which can be opened by going to the about tab
>     and click **Give Feedback**.

Author: ```suggestion
-   Integrated the functionality of the Feedback tab in the **Give Feedback**
    dialog in the About tab.
```
